omaha/main.scons

```diff
@@ -65,10 +65,10 @@ _is_google_update_build = False
 # These values must be changed when customizing open source Omaha.
 # You may also wish to replace the language-neutral constants in the individual
 # *generated_resources_*.rc files with localized strings.
-_FULL_COMPANY_NAME = 'OmahaCompanyName Inc.'
-_SHORT_COMPANY_NAME = 'OmahaCompanyName'
+_FULL_COMPANY_NAME = 'Dental Wings Inc.'
```

Comment:
I would prefer to make it configurable via env variables, like many of the other values already are
`_FULL_COMPANY_NAME = os.getenv('FULL_COMPANY_NAME', 'OmahaCompanyName Inc.')`